src/combobox/ComboBox.ts

```diff
@@ -352,39 +352,39 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 		let controls: DNode = v('div', {
 			classes: this.classes(css.controls)
 		}, [
-			w(TextInput, {
+			w<TextInputProperties>(TextInput, {
 				...inputProperties,
 				bind: this,
 				classes: this.classes(clearable ? css.clearable : null),
 				controls: menuId,
 				disabled,
 				invalid,
-				readOnly,
-				required,
-				value,
 				onBlur: this._onInputBlur,
 				onFocus: this._onInputFocus,
 				onInput: this._onInput,
 				onKeyDown: this._onInputKeyDown,
-				overrideClasses: css
+				overrideClasses: css,
```

Comment:
has changed to `extraClasses`. Plus I think `css` is the wrong thing to pass here